src/vst.cpp

```diff
@@ -159,14 +161,12 @@ VPackBuffer<uint8_t> message::requestHeader(RequestHeader const& header) {
   }
 
   builder.close(); // </array>
-
-  return buffer;
 }
 
-/// @brief creates a slice containing a VST request header.
-VPackBuffer<uint8_t> message::responseHeader(ResponseHeader const& header) {
+/// @brief creates a slice containing a VST response-message header.
+void message::responseHeader(ResponseHeader const& header,
+                             VPackBuffer<uint8_t>& buffer) {
   static std::string const message = " for message not set";
```

Comment:
this static string seems completely unnecessary. it seems to be used only once below, and is then concatened to another const char sequence. So let's remove it here and adjust line 188 to throw the complete message.